test/evergreen/install_gcp_dependencies.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Install dependencies for the GCP SDK. This method is being used instead of having them installed 
+# on system to reduce any versioning problems in the future if the GCP SDK requirements were to
+# change. The installation of the libraries were followed from GCP SDK's documentation:
+# https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md
+#
+install_abseil ()
+{
+    mkdir abseil-cpp && cd abseil-cpp
+    curl -sSL https://github.com/abseil/abseil-cpp/archive/20230125.0.tar.gz -o abseil-cpp.tar.gz
+    tar --strip-components=1 -xzf abseil-cpp.tar.gz
+    sed -i 's/^#define ABSL_OPTION_USE_\(.*\) 2/#define ABSL_OPTION_USE_\1 0/' "absl/base/options.h"
+    mkdir cmake-out && cd cmake-out
+    $CMAKE \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DABSL_BUILD_TESTING=OFF \
+        -DBUILD_SHARED_LIBS=yes ../.
+    make -j 4
+    cd ..
+    sudo /opt/cmake/bin/cmake --build cmake-out --target install
+    cd ..
```

Comment:
And why don't we try to use Ninja too?

Reply:
Thanks for raising this up! There were a few reasons for this going through this approach and getting packaged pre-installed on target machines is something I'd like to look into after this ticket. This is because the time for the BUILD ticket can take quite a long time for turnaround and puts a dependency on the BUILD team if things do not install properly or they start to use a new machine.

Unfortunately the apt-get package doesn't contain the three packages from my understanding (I will do more investigation into this).

Reply:
Sounds good to me - make sure to create a BUILD ticket for reference.

Reply:
How'd you go with this? I'm also happy to discuss this offline.

Reply:
Talked offline created ticket here https://jira.mongodb.org/browse/WT-10651!

Reply:
Thanks Jie!